Makefile

```diff
@@ -229,7 +229,9 @@ em_quarter_ss : wrf
 	( cd test/em_quarter_ss ; /bin/rm -f kernels.asc_s_0_03_0_9 ; ln -s ../../run/kernels.asc_s_0_03_0_9 . )
 	( cd test/em_quarter_ss ; /bin/rm -f kernels_z.asc ; ln -s ../../run/kernels_z.asc . )
 	( cd test/em_quarter_ss ; /bin/rm -f masses.asc ; ln -s ../../run/masses.asc . )
-	( cd test/em_quarter_ss ; /bin/rm -f termvels.asc ; ln -s ../../run/termvels.asc . )
+	( cd test/em_quarter_ss ; /bin/rm -f termvels.asc ; ln -s ../../run/termvels.asc . ; \
+		ln -sf ../../run/RRTM_DATA . ; \
+		ln -sf ../../run/p3* . ) 
```

Comment:
@kkeene44 @davegill The p3* data were not used by the default namelists, perhaps we should not link them.